src/js/components/navigation.js

```diff
@@ -42,6 +44,23 @@ const navigation = behavior({
   [ CLICK ]: {
     [ OPENERS ]: toggleNav,
     [ CLOSERS ]: toggleNav,
+    [ NAV_LINKS ]: function() {
+      // A navigation link has been clicked! We want to collapse any
+      // hierarchical navigation UI it's a part of, so that the user
+      // can focus on whatever they've just selected.
+
+      // Some navigation links are inside accordions; when they're
+      // clicked, we want to collapse those accordions.
+      const acc = this.closest(accordion.ACCORDION);
+      if (acc) {
+        accordion.getButtons(acc).forEach(btn => accordion.hide(btn));
```

Comment:
I'd keep the conditional for the future, it seems like it would make sense for me that it would come up.